README.md

```diff
@@ -1,21 +1,26 @@
 # isaac-api
 
-![Java CI with Maven](https://github.com/isaacphysics/isaac-api/workflows/Java%20CI%20with%20Maven/badge.svg?branch=master)
-[![codecov](https://codecov.io/gh/isaacphysics/isaac-api/branch/master/graph/badge.svg)](https://codecov.io/gh/isaacphysics/isaac-api)
-[![Language grade: Java](https://img.shields.io/lgtm/grade/java/g/isaacphysics/isaac-api.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/isaacphysics/isaac-api/context:java)
+![Java CI with Maven](https://github.com/isaaccomputerscience/isaac-api/workflows/Java%20CI%20with%20Maven/badge.svg?branch=main)
+[![codecov](https://codecov.io/gh/isaaccompterscience/isaac-api/branch/main/graph/badge.svg)](https://codecov.io/gh/isaaccomputerscience/isaac-api)
```

Comment:
Going off the workflow tickets I've seen since I last looked at this PR, I'm not sure we still have codecov any more, I think we're using different coverage tools now.
Still, it's not going to hurt if you want to leave this for now, we can come back to it later.